### PR TITLE
docs: update readme.md

### PR DESCRIPTION
**Pull request by Hannsek**
This pr is editing the `readme.md` file.

Feel free to adjust it more to developer needs.

@Nodonisko Can you add some info regarding suite_mobile?
@mroz22 / @matejkriz Feel free to add broader info to **Connect** / **Suite** section.

### CODE REVIEW

**Comment by mroz22** [comment]
I would personally ditch the 

```
## Packages 
``` 
section completely and replace it with something like "scopes"

```
## Scopes

- packages 
- suite-common
- suite-native
```

@Nodonisko had some nice presentation how these scopes play together.

**Comment by Hannsek** [comment]
Please, edit it as you wish.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,47 @@
 # TREZOR SUITE MONOREPO
 
+Welcome to the Trezor Suite repository! This repository contains the source code for 3 projects:
+
+-   **Connect** - tool that offers developers an easy way to integrate Trezor's functionality into their own applications
+-   **Trezor Suite** - desktop and web application for managing Trezor hardware wallet
+-   **Mobile Suite Lite** - Mobile application designed for tracking user portfolios
```
Comment (tomasklim) [inline README.md]:
```suggestion
-   **Trezor Suite Lite** - Mobile application designed for tracking user portfolios
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,47 @@
...
+
 ![img](https://repository-images.githubusercontent.com/148657224/439f6100-765f-11e9-9bff-b725eef3c4a6)
 
+# Development
+
+### Prerequisities
+
+-   [NVM](https://github.com/nvm-sh/nvm)
+-   [Yarn](https://yarnpkg.com/lang/en/docs/install/)
+-   [git lfs](https://git-lfs.github.com/)
```
Comment (tomasklim) [inline README.md]:
```suggestion
-   [Git LFS](https://git-lfs.github.com/)
```

**Review by matejkriz — APPROVED** [review]
I believe this is an improvement and we can merge it, of course there is a lot of room for further improvement and anyone can open another PR at any time... For example it would be nice to describe the scopes as @mroz22  mentioned.

**Comment by komret** [comment]
Is it intended to ditch the package descriptions without replacement?

**Comment by Nodonisko** [comment]
@matejkriz I don't want to define it until desktop app will have it's own folder otherwise scopes doesn't make that much sense and it will be very hard to describe them because of exceptions.

**Comment by matejkriz** [comment]
would be nice to describe scopes, would be nice to separate desktop&web from other packages so it's easier to explain, but I believe that even just removing incomplete package descriptions is improvement.

**Pull request merged**